src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -487,8 +491,8 @@ void LiveFunctionsDataView::OnTimer() {
 
 void LiveFunctionsDataView::OnRefresh(const std::vector<int>& visible_selected_indices,
                                       const RefreshMode& mode) {
-  if (mode != RefreshMode::kOnFilter && mode != RefreshMode::kOnSort) return;
-  UpdateHighlightedFunctionId(visible_selected_indices);
+  if (mode != RefreshMode::kOther) UpdateHighlightedFunctionId(visible_selected_indices);
```

Comment:
We specified the `kOnFilter` and `kOnSort` modes in the if condition on purpose. This was mainly to highlight that we will update `highlighted_function_id` while sorting or filtering.

So I prefer
```cpp
if (mode == RefreshMode::kOnFilter || mode == RefreshMode::kOnSort){
   UpdateHighlightedFunctionId(visible_selected_indices);
}
```
here.